docs/protocols.md

```diff
@@ -27,10 +26,16 @@ If stronger algorithms are needed (for example, AES with 256-bit keys), the [JCE
 
 Protocol uses non-blocking I/O wherever possible which removes the performance bottleneck of the <code>JNLP3-connect</code> protocol.
 
+### WebSocket
+
+* Introduced in: Remoting version TODO, [JEP-222](https://jenkins.io/jep/222)
```

Comment:
Should it firstly be released with "Beta" disclaimers until there is a community feedback and JEP being accepted?

Reply:
Makes sense. I think that can be inserted by the merger when inserting the actual version number here?

Reply:
I need to remember to do that.

Reply:
What sort of beta disclaimers do we want? Just a brief comment that this capability is considered beta until further notice?

Reply:
Sounds right.

Reply:
Going to do that as a follow-up commit or what?

Reply:
Apparently. I missed my note to merge it in with the other. I'll just do it now.